src/styles/elements/buttons.scss

```diff
@@ -49,6 +63,11 @@
             &:hover {
                 color: $argo-color-gray-7;
             }
+
+            &:focus {
+                color: $argo-color-gray-7;
+                border: 1px solid $argo-color-teal-5 !important;
```

Comment:
Does this style not work without the `!important`? I like to avoid using `!important` if possible

Reply:
Trying to test it and it's actually not working with or without the `!important`. I think that disabled buttons are always skipped over in the DOM and never get focus. I've removed this entire pseudoclass.